scripts/slack-release-notification.mjs

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env zx
+
+import SlackNotify from 'slack-notify';
+import {config} from 'dotenv';
+import { createRequire } from "module"; 
+
+const require = createRequire(import.meta.url); 
+const json = require("../packages/components/package.json");
+
+const result = config({path: 'local.env'});
+const slack = SlackNotify(result.parsed.SLACK_NOTIFICATION);
+
+const message = ':information_source: New Release v' + json.version + ' :tada:';
+
+slack.send({
```

Comment:
kannst du den block von Promises in `async`/`await` umschreiben? :)

https://stackoverflow.com/questions/55272701/converting-promises-to-async-await-javascript